resource_management_mobile/src/app/auth/login/login.page.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { LoginService } from '../login.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { LoginRequest, LoginResponse } from '../models/auth.model';
+import { Router } from '@angular/router';
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.page.html',
+  styleUrls: ['./login.page.scss'],
+})
+export class LoginPage implements OnInit {
+  loginForm !: FormGroup;
+  showPasswordText:boolean = false;
+  type:string = 'password';
```

Comment:
type is not valid name - use passwordInputType

Reply:
variable name changed..